doc/using/overlays.chapter.md

```diff
@@ -1,69 +1,166 @@
 # Overlays {#chap-overlays}
 
-This chapter describes how to extend and change Nixpkgs using overlays.  Overlays are used to add layers in the fixed-point used by Nixpkgs to compose the set of all packages.
+This chapter describes how to extend and change Nixpkgs using overlays.
 
-Nixpkgs can be configured with a list of overlays, which are applied in order. This means that the order of the overlays can be significant if multiple layers override the same package.
+TODO: Maybe add link to `lib.extends`/`lib.fix`
 
-## Installing overlays {#sec-overlays-install}
+## What are overlays {#sec-overlays-definition}
```

Comment:
I think the important question is "what can i do with overlays". The technical correct definition is not as important in the first sentence.

```suggestion
## What are overlays {#sec-overlays-definition}

Overlays can be used in general to extend or change existing attribute sets. Common applications are:

- Adding/extending packages to a locally imported version of nixpkgs
- Overriding packages
- Overriding packageSets

```